documentation/src/main/wiki/ClickModifier.md

```diff
@@ -11,7 +11,8 @@ Module name: **click**
 ##### Parameters
 | Parameter | Default value | Description | Mandatory |
 | --------- | ------------- | ----------- | --------- |
-| `xpath` | | xpath of the element to be clicked | yes |
+| `xpath` | | xpath of the element to be clicked        | yes |
```

Comment:
Please mention that only one selector may be used in each `click` modifier entry (maybe it is wroth to mention the obvious, that there may be some selectors with `xpath` and some with `css` in the same test).